<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FriendForm</class>
 <widget class="QWidget" name="FriendForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>340</width>
    <height>80</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>340</width>
    <height>80</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>340</width>
    <height>80</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="pb_icon">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>60</width>
     <height>0</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="iconSize">
    <size>
     <width>60</width>
     <height>60</height>
    </size>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="lb_name">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>6</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>mmm</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_feeling">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>40</y>
     <width>120</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>12</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
